src/feed/Feed.ts

```diff
@@ -599,7 +630,10 @@ class Feed {
                 tick => CIQ.strToDateTime(tick.Date) >= CIQ.strToDateTime(getUTCDate(this.endEpoch as number))
             );
             if (endTickIndex > -1) {
-                const addon = trimmedQuotes[endTickIndex].Date === getUTCDate(this.endEpoch) ? 2 : 1;
+                const addon =
+                    CIQ.strToDateTime(trimmedQuotes[endTickIndex].Date) === CIQ.strToDateTime(getUTCDate(this.endEpoch))
```

Comment:
fix against exit tick being skipped (not drawn on the chart) as a result of the wrong `addon` value.